### PR TITLE
Xamarin.Forms Augmented reality samples

### PR DESCRIPTION
**Pull request by ZackAllen**

### CODE REVIEW

File: src/Forms/AugmentedReality/Samples/NavigateAR/RoutePlanner.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+
+        #region Authentication
+
+        // Constants for OAuth-related values.
+        // - The URL of the portal to authenticate with (ArcGIS Online).
+        private const string ServerUrl = "https://www.arcgis.com/sharing/rest";
+        // - The Client ID for an app registered with the server (the ID below is for a public app created by the ArcGIS Runtime team).
+        private const string AppClientId = @"lgAdHkYZYlwwfAhC";
+        // - A URL for redirecting after a successful authorization (this must be a URL configured with the app).
+        private const string OAuthRedirectUrl = @"my-ags-app://auth";
```
[the remark this answers is not in the document]
Comment (cgutierrez-gis) [reply src/Forms/AugmentedReality/Samples/NavigateAR/RoutePlanner.xaml.cs]: I've noticed this behavior during certification for several other samples requiring a login of some sort. I can provide a list of samples that i've seen are affected if needed @ZackAllen

[the remark this answers is not in the document]
Comment (ZackAllen) [reply src/Forms/AugmentedReality/Samples/NavigateAR/RoutePlanner.xaml.cs]: Fixed in Navigate AR, and I created an issue for the other samples.

File: src/Forms/iOS/ArcGISRuntime.Xamarin.Forms.iOS.csproj [hunk 2]
```diff
@@ -198,35 +198,41 @@
     <BundleResource Include="Resources\Info%402x.png" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Esri.ArcGISRuntime.ARToolkit.Forms">
+      <Version>100.7.1</Version>
```
[the remark this answers is not in the document]
Comment (ZackAllen) [reply src/Forms/iOS/ArcGISRuntime.Xamarin.Forms.iOS.csproj:202]: I did not need to reference this on my mac.

[the remark this answers is not in the document]
Comment (cgutierrez-gis) [reply src/Forms/iOS/ArcGISRuntime.Xamarin.Forms.iOS.csproj:202]: I didn't either.

**Pull request merged**